services/blockchain-indexer/methods/dataService/blocks.js

```diff
@@ -27,13 +28,25 @@ module.exports = [
 			blockID: { optional: true, type: 'any' },
 			height: { optional: true, type: 'any' },
 			generatorAddress: { optional: true, type: 'any' },
-			generatorPublicKey: { optional: true, type: 'any' },
 			timestamp: { optional: true, type: 'any' },
 			limit: { optional: true, type: 'any' },
 			offset: { optional: true, type: 'any' },
 			sort: { optional: true, type: 'any' },
 		},
 	},
+	{
+		name: 'blocks.assets',
+		controller: getBlocksAssets,
+		params: {
+			blockID: { optional: true, type: 'string' },
+			height: { optional: true, type: 'string' },
+			moduleID: { optional: true, type: 'string' },
+			timestamp: { optional: true, type: 'string' },
+			limit: { optional: true, type: 'number' },
+			offset: { optional: true, type: 'number' },
+			sort: { optional: true, type: 'string' },
+		},
+	},
 	{
 		name: 'blocks.last',
```

Comment:
Do we use this method anywhere? If not, please remove it from the dataService layer.